src/hw05_01_runner.py

```diff
@@ -0,0 +1,29 @@
+"""
+1.	Оформите решение задач из прошлых домашних
+работ в функции. Напишите функцию runner.
+a.	runner() – все фукнции вызываются по очереди
+b.	runner(‘func_name’) – вызывается только функцию func_name.
+c.	runner(‘func’, ‘func1’...) - вызывает все переданные функции
+
+"""
+
+
+import mymodule
+
+
+def runner(*args):
+    if not args:
+        args = [elements for elements in
```

Comment:
we should also check if it's callable

Reply:
implemented a function check for existence